README.md

```diff
@@ -1,2 +1,97 @@
-# steampipe-mod-office365-compliance
-Run individual controls or full compliance benchmarks for CIS across all of your Azure active directories using Steampipe.
+# Microsoft 365 Compliance Mod for Steampipe
+
+83+ checks covering industry defined security best practices for Microsoft 365. Includes full support for `CIS v1.4` compliance benchmarks across all of your Microsoft 365 tenants.
```

Comment:
@khushboo9024 Can you please update this with the new number now that we've removed manual controls?